common/djangoapps/third_party_auth/models.py

```diff
@@ -212,6 +212,16 @@ class ProviderConfig(ConfigurationModel):
         default=False,
         help_text="Use the presence of a profile from a trusted third party as proof of identity verification.",
     )
+
```

Comment:
given the testing you are doing in enterprise already probably not but , do you see a need to test this field in any way in edx-platform? like if we remove this the test would fail, I guess

Reply:
and I also just saw your note in the desc, sorry... 
> Verifying the field exists in Django Admin simply requires running the migration here, ensuring LMS has restarted, and then opening the ProviderConfig form in Django Admin.

Reply:
Yeah that was just manual testing notes though. I think your question relates to automated testing, and to answer that: I don't really see a need to add a test here. The only way an admin or model level test would fail is if someone explicitly removed this field at which point...well of course the tests fail.